internal/backends/selector/Cargo.toml

```diff
@@ -19,6 +19,9 @@ backend-gl-all = ["backend-gl-x11", "backend-gl-wayland"]
 backend-gl-wayland = ["i-slint-backend-gl/wayland"]
 backend-gl-x11 = ["i-slint-backend-gl/x11"]
 
+rtti-gl = ["i-slint-backend-gl/rtti"]
+rtti-qt = ["i-slint-backend-qt/rtti"]
```

Comment:
Weak features will be stabilized with rust 1.60

Reply:
Ok, in that case I think I'll close this merge request again - this is messy and gets worse for the C++ crate (as the CI indicates).

Reply:
why is it messy?  i only see one line that is not gated

Reply:
True, that was missing. I *think* the C++ crate build will fail though, because nothing is enabling rtti.

Reply:
The interpreter should enable it

Reply:
How can the interpreter do that without a weak feature?

The C++ crate needs to select the backend directly via `i-slint-backend-selector/backend-XXX`. It can't select the backend via the `interpreter` crate because the interpreter is optional, i.e. it should be possible to build `slint-cpp`, select a backend but skip the interpreter.

That's how I ended up with the `rtti-qt` and `rtti-gl` features in the C++ crate, as CMakeLists.txt can select those if the interpreter is selected /and/ `gl` or `qt` respectively is.